projects/plugins/jetpack/modules/google-fonts/current/load-google-fonts.php

```diff
@@ -148,6 +150,106 @@ function ( $google_fonts_family ) use ( $available_google_fonts_map, $theme_font
 
 add_filter( 'wp_theme_json_data_default', 'jetpack_register_google_fonts_to_theme_json' );
 
+/**
+ * Filter out the deprecated font families that are from the jetpack-google-fonts provider.
+ *
+ * @param object[] $font_families The font families.
+ * @return object[] The filtered font families.
+ */
+function jetpack_google_fonts_filter_out_deprecated_font_data( $font_families ) {
+	return array_values(
+		array_filter(
+			$font_families,
+			function ( $font_family ) {
+				$has_deprecated_google_fonts_data = false;
+				foreach ( $font_family['fontFace'] as $font_face ) {
+					$provider = isset( $font_face['provider'] ) ? $font_face['provider'] : '';
```

Comment:
Nit: why not
```suggestion
					$provider = $font_face['provider'] ?? '';
```

Reply:
The null coalescing operator (??) was introduced in PHP 7 but the Jetpack plugin supports [5.6 or higher](https://wordpress.org/plugins/jetpack/). I'm unsure whether this feature is available on the client site.